src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepositoryHelper.java

```diff
@@ -1037,6 +1037,9 @@ private void makeRepoOrphan(Repository repository, String site) throws IOExcepti
             logger.debug("Soft reset to commit empty repo");
             git.reset().call();
 
+            // Include all files in the first commit
+            git.add().addFilepattern(GIT_COMMIT_ALL_ITEMS).call();
```

Comment:
It will cause craftercms/craftercms#3549.
We need to have the empty repository commit first.
Please find another way to resolve your issue.